sim/aviron/src/lib/io.zig

```diff
@@ -120,12 +122,18 @@ pub const RAM = struct {
 
             fn mem_read(ctx: ?*anyopaque, addr: Address) u8 {
                 const mem: *Self = @ptrCast(@alignCast(ctx.?));
-                return mem.data[addr];
+                // ATmega328P memory map: SRAM starts at 0x0100
+                // Map logical addresses 0x0100-0x08FF to array indices 0x0000-0x07FF
+                const sram_offset: Address = if (addr >= 0x100) addr - 0x100 else addr;
```

Comment:
I would rather make this a configurable option, as it's part of a *Static* memory, and not a ATmega328P memory. This seems to be incredibly unexpected.

Reply:
Yes, I have another change that lets memory ranges to take bases, which is configurable